src/vs/editor/contrib/codeAction/browser/codeActionMenu.ts

```diff
@@ -30,6 +30,18 @@ import { IKeybindingService } from 'vs/platform/keybinding/common/keybinding';
 import { ResolvedKeybindingItem } from 'vs/platform/keybinding/common/resolvedKeybindingItem';
 import { ITelemetryService } from 'vs/platform/telemetry/common/telemetry';
 import { IThemeService } from 'vs/platform/theme/common/themeService';
+import 'vs/base/browser/ui/codicons/codiconStyles'; // The codicon symbol styles are defined here and must be loaded
+import 'vs/editor/contrib/symbolIcons/browser/symbolIcons'; // The codicon symbol colors are defined here and must be loaded to get colors
+import { Codicon } from 'vs/base/common/codicons';
+
+
+export enum CodeActionGroupID {
```

Comment:
You should be able to use the existing `CodeActionKind` types for this (adding new ones if needed)